docs/local-development.md

```diff
@@ -515,7 +515,7 @@ We shallow render the component and simulate an UI interaction like `click`.
 
 ```js
 it("should call handleClick function", () => {
-  const onClick = jest.genMockFunction();
+  const onClick = jest.fn();
```

Comment:
`genMockFunction` was removed and was an undocumented API. Removing it from our examples and code.

https://github.com/facebook/jest/pull/6173